map/place_data.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include "base/string_utils.hpp"
```

Comment:
The order of groups should be following:
* search
* indexer
* editor
* platform
* base
* std
* 3party